templates/landingPage.html

```diff
@@ -4,7 +4,7 @@
     <title>CSESoc Website | {{index . "title"}}</title>
   </head>
   <body>
-    <h1>{{index . "message"}}</h1>
+    <h1>{{index . "message"}}<br></h1>
```

Comment:
Sorry should have clarified not a `<br>`, your HTML file (and all files) should end with a newline (press return/enter) - the review comment indicates what part of the file I'm talking about